### PR TITLE
Ops caching: fix a bug, add logging

### PR DESCRIPTION
**Pull request by vladsud**

### CODE REVIEW

**Review by jatgarg — APPROVED** [review]

File: packages/drivers/odsp-driver/src/contracts.ts [hunk 1]
```diff
@@ -242,6 +242,7 @@ export interface IOpsCachingPolicy {
      * At the same time, big number means we wait for so many ops to accumulate, which
      * increases chances and number of trailing ops that would not be flushed to cache
      * when user closes tab
+     * Use any number below 1 to disable caching
```
Comment (anthony-murphy) [inline packages/drivers/odsp-driver/src/contracts.ts:245]: would be good to add this case in packages\test\test-drivers\src\odspDriverApi.ts:

```
export const odspOpsCaching: OptionsMatrix<IOpsCachingPolicy> = {
    batchSize: [undefined, -1],
    timerGranularity: numberCases,
    totalOpsToCache: numberCases,
};
```

Comment (vladsud) [reply packages/drivers/odsp-driver/src/contracts.ts:245]: done

File: packages/drivers/odsp-driver/src/opsCaching.ts [hunk 2]
```diff
@@ -96,6 +99,7 @@ export class OpsCache {
 
             this.totalOpsToCache--;
             if (this.totalOpsToCache === 0) {
+                this.logger.sendPerformanceEvent({ eventName: "CacheOpsLimitHit"});
```
Comment (markfields) [inline packages/drivers/odsp-driver/src/opsCaching.ts:102]: @vladsud why are you putting this in the Performance table instead of the Generic table?

Comment (vladsud) [reply packages/drivers/odsp-driver/src/opsCaching.ts:102]: It has impact on perf. Note - semantic here - what is the category of this event (there is no notion of tables here)
We can put it anywhere, but I find it more convenient for all perf-related events to be in one place

**Pull request merged**